modules/docs/src/Volo.Docs.EntityFrameworkCore/Volo/Docs/Projects/EfCoreProjectRepository.cs

```diff
@@ -39,5 +38,10 @@ public async Task<Project> GetByShortNameAsync(string shortName)
 
             return project;
         }
+
+        public async Task<bool> ShortNameExistsAsync(string shortName)
+        {
+            return await DbSet.AnyAsync(x => x.ShortName == shortName);
```

Comment:
This comparison should be case sensitive.

Reply:
Ef core does not seem to support such a function. In addition, sqlserver is not case sensitive but not all databases are like this. So we should normalize ShortName.